metricbeat/module/php_fpm/process/process_integration_test.go

```diff
@@ -23,10 +23,28 @@ import (
 	"os"
 	"testing"
 
+	"github.com/stretchr/testify/assert"
+
 	"github.com/elastic/beats/libbeat/tests/compose"
 	mbtest "github.com/elastic/beats/metricbeat/mb/testing"
 )
 
+func TestFetch(t *testing.T) {
+	compose.EnsureUp(t, "phpfpm")
+
+	f := mbtest.NewReportingMetricSetV2(t, getConfig())
+	events, errs := mbtest.ReportingFetchV2(f)
+
+	assert.Empty(t, errs)
+	if !assert.NotEmpty(t, events) {
+		t.FailNow()
+	}
+
+	t.Logf("%s/%s event: %+v", f.Module().Name(), f.Name(),
+		events[0].BeatEvent("haproxy", "info").Fields.StringToPrint())
```

Comment:
Not sure I understand this mention of `haproxy` here. The log should say "php_fpm/process", right?

Reply:
yes